src/test/osdc/object_cacher_stress.cc

```diff
@@ -47,7 +47,7 @@ class C_Count : public Context {
     : m_op(op), m_outstanding(outstanding) {}
   void finish(int r) override {
     m_op->done++;
-    assert(m_outstanding > 0);
+    assert(*m_outstanding > 0);
     *m_outstanding--;
```

Comment:
ceph/src/test/osdc/object_cacher_stress.cc: In member function ‘virtual void C_Count::finish(int)’:
ceph/src/test/osdc/object_cacher_stress.cc:51:5: warning: value computed is not used [-Wunused-value]
     *m_outstanding--;